concent_api/core/transfer_operations.py

```diff
@@ -297,14 +297,14 @@ def calculate_token_expiration_deadline(
 ) -> int:
     if operation == FileTransferToken.Operation.upload:
         token_expiration_deadline = (
-            report_computed_task.task_to_compute.compute_task_def['deadline'] +
+            int(report_computed_task.task_to_compute.compute_task_def['deadline']) +
             3 * settings.CONCENT_MESSAGING_TIME +
             2 * calculate_maximum_download_time(report_computed_task.size, settings.MINIMUM_UPLOAD_RATE)
         )
 
     elif operation == FileTransferToken.Operation.download:
         token_expiration_deadline = (
-            report_computed_task.task_to_compute.compute_task_def['deadline'] +
+            int(report_computed_task.task_to_compute.compute_task_def['deadline']) +
             calculate_subtask_verification_time(report_computed_task)
         )
     return token_expiration_deadline
```

Comment:
Quite frankly, casting could be done here (in one pleace)

Reply:
`deadline` could also be a `string` type, so we need that conversion for operation to work.

Reply:
my bad, I missed that